### DISCUSSION TITLE
[1.6.1] 2022-05-26

### DISCUSSION THREAD

**Discussion by darthcloud** [darthcloud/BlueRetro#288] [General]
# Global
* Fix memory card read on Android (Closes #277)

<hr /><em>This discussion was created from the release <a href='https://github.com/darthcloud/BlueRetro/releases/tag/v1.6.1'>[1.6.1] 2022-05-26</a>.</em>

**Comment by khtp** [comment]
This hapened 
![Uploading 20220527_004049.jpg…]()
The same thing keeps happening to me but this time it doesn't show me a message that flashes, that is, the message "control disconnected" but in seconds and it detects it again and out of nowhere the control doesn't react, I have to restart the esp32 so that it detects it again, I left it on pause for 30 minutes and when I returned the control it had turned off and when I tried to turn it on it no longer wanted to react in the game and I had to restart esp32 to detect it again, it worked for 1 hour, I'm going to upload the video on drive and I'll pass it to you so you can see what happens, it happens to me on ps2 as well as on wii configured as gamecube I'm kadher torres

**Reply by darthcloud** [reply]
Use your existing thread #248 for future comments
